src/core/main.c

```diff
@@ -1390,8 +1390,11 @@ static int bump_unix_max_dgram_qlen(void) {
          * $NOTIFY_SOCKET one. */
```

Comment:
not that i matters much, but we are not doing `Signed-off-by` btw, that's a kernel thing

Reply:
dropped